src/UIAutomation/Microsoft.UI.UIAutomation/Client.g.cpp

```diff
@@ -3283,6 +3283,16 @@ namespace winrt::Microsoft::UI::UIAutomation::implementation
         return As<AutomationRemoteTextRange>();
     }
 
+    winrt::AutomationRemoteBool AutomationRemoteAnyObject::IsConnectionBoundObject()
+    {
+        throw hresult_not_implemented();

```

Comment:
Will this be implemented in the future?

Reply:
Otherwise, I'd say leave a comment why it isn't implemented?

Reply:
OK, I looked into this, and I think you can just remove this? It makes sense to me that since connection bound objects don't have a concrete type in the remote op VM, they don't need a method like this. If it's not implemented, we might as well not have the method.